### ISSUE TITLE
Internal: The `verify-bundles` freeze's helper script fails.

### ISSUE THREAD

**Issue by jansiegel** [handsontable/handsontable#9891]
### Description

The `script/verify-bundles.mjs` script fails with the following error when trying to use the `npm run freeze` script.

```
node:internal/errors:477
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "/workspace/handsontable/package.json" needs an import assertion of type "json"
    at new NodeError (node:internal/errors:387:5)
    at validateAssertions (node:internal/modules/esm/assert:82:15)
    at defaultLoad (node:internal/modules/esm/load:84:3)
    at nextLoad (node:internal/modules/esm/loader:165:28)
    at ESMLoader.load (node:internal/modules/esm/loader:608:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:464:22)
    at new ModuleJob (node:internal/modules/esm/module_job:63:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:483:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:441:34)
    at async Promise.all (index 0) {
  code: 'ERR_IMPORT_ASSERTION_TYPE_MISSING'
}
```

This is most probably caused by the changes introduced in https://github.com/handsontable/handsontable/pull/9267.

### Steps to reproduce

1. From the root directory, run `node --experimental-json-modules ./scripts/verify-bundles.mjs`
2. ... or run `npm run freeze` and wait until it calls the `verify-bundles` script.

**Comment by warpech** [comment]
@jansiegel I think that about half of `verify-bundles` has the potential to be reused in https://github.com/handsontable/handsontable/issues/9879. If you agree, what lines would you recommend? Maybe this script can be changed to accept a parameter that would run `verify-bundles` in a prerelease mode?

**Comment by jansiegel** [comment]
@warpech Yup, good idea - after modifying it to accept an argument, the entirety of the script can be reused, I think. All it does is verifies if the classes exported from the packages contain the correct version number, which could potentially be useful in #9879.

**Comment by AMBudnik** [comment]
Closed as done in Handsontable v12.3.0.
